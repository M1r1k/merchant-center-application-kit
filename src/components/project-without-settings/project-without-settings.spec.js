import React from 'react';
import { shallow } from 'enzyme';
import ProjectWithoutSettings from './project-without-settings';

describe('rendering', () => {
  let wrapper;
  beforeEach(() => {
    wrapper = shallow(
      <ProjectWithoutSettings params={{ currentProject: 'foo' }} />
    );
  });
  it('outputs correct tree', () => {
    expect(wrapper).toMatchSnapshot();
  });
});
