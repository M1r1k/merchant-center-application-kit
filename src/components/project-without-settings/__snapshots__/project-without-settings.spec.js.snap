// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`rendering outputs correct tree 1`] = `
<div>
  Unexpected error. The project-setting is missing for project foo. Please contact our support.
</div>
`;
